include/pidomus.h

```diff
@@ -442,13 +442,25 @@ class piDoMUS : public ParameterAcceptor, public SundialsInterface<typename LAC:
   unsigned int max_iterations;
 
   /**
+   * Maximum number of temprary vectors used by FGMRES for solving
```

Comment:
temporary. I would omit `for solving the Newton's system`. I lean to something like `FGMRES for the solution of the linear system using the coarse preconditioner.`